### PR TITLE
WIP: changes to make dall-e-worker example in iroh-examples work

### PR DESCRIPTION
**Pull request by b5**

### CODE REVIEW

File: src/doc.rs [hunk 1]
```diff
@@ -51,6 +51,36 @@ impl Doc {
...
+            let hash = self
+                .inner
+                .set_bytes(author_id.0, key, value)
+                .await
+                .map_err(IrohError::doc)?;
+            Ok(Arc::new(Hash(hash)))
+        })
+    }
+
+    pub fn get_content_bytes(&self, entry: Arc<Entry>) -> Result<Vec<u8>, IrohError> {
```
Comment (ramfox) [inline src/doc.rs:72]: you can call `read_to_bytes` for this same functionality

File: src/doc.rs [hunk 2]
```diff
@@ -51,6 +51,36 @@ impl Doc {
         })
     }
 
+    pub fn set_file_bytes(
```
Comment (ramfox) [inline src/doc.rs:54]: working on this right now on iroh api `Doc::import_file`

File: go/iroh/iroh.go [hunk 3]
```diff
@@ -1,6 +1,10 @@
 package iroh
 
-// #include <iroh.h>
+/*
+#cgo CFLAGS: -I../../target/release
+#cgo LDFLAGS: -liroh -L../../target/release
+#include <iroh.h>
+*/
```
Comment (ramfox) [inline go/iroh/iroh.go:7]: did you add this for convenience, or was this generated? Should we add this?

File: make_go.sh [hunk 4]
```diff
@@ -1,5 +1,5 @@
 set -eu
 
-cargo build
+cargo build --release
```
Comment (ramfox) [inline make_go.sh:3]: assuming this was in coordination w/ adding the `cgo` flags stuff. should we keep this?

**Comment by b5** [comment]
stale. closing

**Pull request closed without merging**